org.gtaf.quran.appdata.xml

```diff
@@ -0,0 +1,218 @@
+            <br />
+            ● Surah Search along with advanced search tips to quickly find what you are looking for
+            <br />
+            ● Explore by Topics &amp; read all ayahs related to a topic together
+            <br />
+
+        </p>
+        <p>
+            <strong>Quran Audio</strong>
+            <br />
+            <br />
+            ● Listen to several recitations by 30+ reciters (downloadable for offline use)
+            <br />
+            ● Reciter Options: Sheikh Mishary Al Afasy, Sheikh Husary (Muallim), Shaykh Ayman
+            Suwaid,
+            Sheikh Abdur Rahman As-Sudais and many others
+            <br />
+            ● Robust Audio System with repeat, group playback of verses function to aid in Quran
+            Memorization / Quran Hifz
+            <br />
+            ● Reciters tagged based on recitation types: Murattal, Mujawwad, WBW, Translation
+            <br />
+            ● English Quran Audio Translation and Arabic Audio Commentary
+        </p>
+        <p>
+            ● Word by Word Audio Playback
+
+            <br />
+            <br />
+
+            <strong>Various Customization Options &amp; Others</strong>
+            <br />
+            <br />
+            ● Read in Uthmanic / Indopak script
+            <br />
+            ● Read Tafsirs in Tafsir view
+            <br />
+            ● Recite Tajweed colour-coded Quran with ease
+            <br />
+            ● Quran Dictionary
+            <br />
+            ● Various fonts and multiple themes, including Night mode
+            <br />
+            ● Autoscroll feature
+            <br />
+            ● Copy &amp; Share the verses
+            <br />
+        </p>
+        <p>
+            <strong>Localization</strong>
+            <br />
+            <br />
+            ● Quran App User Interface localized in 20+ Languages: Arabic, English, Bangla, German,
+            French, Bahasa Indonesia / Malay, Russian, Spanish, Tagalog, Turkish, Urdu, and more
+            <br />
+
+            ● All feature supports offline (Quran offline)
+            <br />
+            ● And the next features that are in development
+
+            <br />
+            <br />
+            Download now and start your journey to a deeper understanding of the Quran!
+
+            <br />
+            <br />
+            Share and recommend this beautiful Quran app to your friends and family. May Allah bless
+            us
+            in this world and hereafter.
+
+            <br />
+            <br />
+            "Whoever calls people to right guidance will have a reward like that of those who follow
+            him..." - Sahih Muslim, Hadith 2674
+
+            <br />
+        </p>
+        <p>
+            Developed by
+            <strong>Greentech Apps Foundation</strong>
+            <br />
+            <br />
+            Website:
+            <a href="https://gtaf.org">https://gtaf.org</a>
```

Comment:
href are NOT allowed in the description.

Reply:
Fixed